drivers/linux/chipsec_km.c

```diff
@@ -702,10 +705,9 @@ void * patch_read_msr(void * CPUInfo)
 	return NULL;
 }
 
-
+#ifdef EFI_NOT_READY
```

Comment:
Not sure why the chose EFI_NOT_READY as the magic value to check to see if efi.h was included.  Makes for some odd code to read.

Reply:
That code was already within the driver I just moved it above the function given everything it does depends upon efi functionality.

Reply:
Understood, just commenting on the original choice.  Nothing for you to change around this code.